src/packages/@ncigdc/components/SearchPage.js

```diff
@@ -100,7 +100,7 @@ const SearchPage = (
         )}
         <CurrentFilters style={{ flex: 1 }} {...filtersLinkProps} />
       </Row>
-      {results}
+      {typeof results === 'function' ? results({ showFacets }) : results}
```

Comment:
i think this might have reverted back to pre-`1.23.0` code, `release` branch just has `{results}`

Reply:
I use this function because in summary page, I need this `showFacets` value to determine whether to show 3 or 4 cards in a row. I can do it in redux too. But it will be more complicated.

Reply:
ok!